contracts/ExecutionManager.sol

```diff
@@ -90,10 +118,23 @@ contract ExecutionManager is IExecutionManager, AccessControlExtended {
      * @return Minimum fees required for executing the transaction
      */
     function getMinFees(
-        uint256,
+        uint256 gasLimit_,
+        uint256 msgValue_,
+        uint256 payloadSize_,
+        bytes32 extraParams_,
```

Comment:
extraParams_ is not used anywhere from what I see.
The intent was to decode msgValue_ from extraParams_ and not have it as a dedicated input.